src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -111,6 +111,10 @@ private function extractXliff1(\DOMDocument $dom, MessageCatalogue $catalogue, s
                     continue;
                 }
 
+                if (isset($translation->target) && 'needs-translation' === (string) $translation->target->attributes()['state']) {
```

Comment:
This would deserve a changelog entry IMO, while sneaking it in the normalization PR makes it invisible.

Reply:
See https://github.com/symfony/symfony/issues/54541